system_test/common/helpers/aest_docker.erl

```diff
@@ -73,7 +74,8 @@
     config_guest_path => nonempty_string(),
     config => #{atom() => term()},
     % Tuple of host/guest paths where the node DB is meant to be if persisted
-    db_path => {binary(), binary()} | undefined
+    db_path => {binary(), binary()} | undefined,
+    entrypoint => [list()] | undefined
```

Comment:
Optional. Ok. If optional, is `undefined` needed? And may I suggest a more specific type for hinting the user/developer:
```suggestion
    entrypoint => [string(), ...]
```
----
I see underlying API is [this](https://docs.docker.com/engine/api/v1.39/#operation/ContainerCreate):
> Entrypoint 
> Array of string
> The entry point for the container as a string or an array of strings.
> If the array consists of exactly one empty string (`[""]`) then the entry point is reset to system default (i.e., the entry point used by docker when there is no `ENTRYPOINT` instruction in the `Dockerfile`).